woocommerce/woocommerce
# CYS: Fix crash homepage

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR adds a check to ensure that the app doesn't crash when ```currentTemplate``` is still undefined.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Visit ```/wp-admin/admin.php?page=wc-admin&customizing=true&path=%2Fcustomize-store%2Fassembler-hub%2Fhomepage```.
2. Ensure that the Assembler is loaded correctly.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

CYS: Fix crash homepage

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
